src/poliastro/coordinates.py

```diff
@@ -130,3 +131,36 @@ def inertial_body_centered_to_pqw(r, v, source_body):
     v_pqw = (np.array([-sin(nu), (ecc + cos(nu)), 0]) * sqrt(k / p)).T * u.km / u.s
 
     return r_pqw, v_pqw
+
+
+def change_frame(orbit, frame_orig, frame_dest):
+    """Transforms Orbit from one frame to another.
+    
+    Parameters
+    ----------
+    orbit : ~poliastro.bodies.Orbit
+        Orbit to transform
+    frame_orig : ~astropy.coordinates.BaseRADecFrame
+        Initial frame
+    frame_dest : ~astropy.coordinates.BaseRADecFrame
+        Final frame
+
+    Returns
+    -------
+    orbit: ~poliastro.bodies.Orbit
+        Orbit in the new frame
+
+    """
+
+    orbit_orig = frame_orig(x=orbit.r[0], y=orbit.r[1], z=orbit.r[2],
+                            v_x=orbit.v[0], v_y=orbit.v[1], v_z=orbit.v[2],
+                            representation=CartesianRepresentation,
+                            differential_type=CartesianDifferential)
+
+    orbit_dest = orbit_orig.transform_to(frame_dest(obstime=orbit.epoch))
+    orbit_dest.representation = CartesianRepresentation
+
+    return Orbit.from_vectors(orbit.attractor,
+                              [orbit_dest.x, orbit_dest.y, orbit_dest.z] * u.km,
+                              [orbit_dest.v_x, orbit_dest.v_y, orbit_dest.v_z] * (u.km / u.s),
```

Comment:
And here I guess it would be `v_xyz` http://docs.astropy.org/en/stable/api/astropy.coordinates.CartesianDifferential.html#astropy.coordinates.CartesianDifferential.d_xyz (not `d_xyz`, see https://github.com/astropy/astropy/pull/6569/files#diff-1e15736771f2d9c393cd740f79ae039f)

Reply:
same thing

Reply:
In this case it's more complex, but it's still there:

```
In [31]: dv = GCRS(x=1 * u.km, y=0 * u.km, z=0 * u.km, v_x=0*u.km/u.s, v_y=1*u.k
    ...: m/u.s, v_z=0*u.km/u.s, representation=CartesianRepresentation, differen
    ...: tial_type=CartesianDifferential)

In [32]: dv.data.xyz
Out[32]: <Quantity [1., 0., 0.] km>

In [33]: dv.data.differentials['s'].d_xyz
Out[33]: <Quantity [0., 1., 0.] km / s>
```